tests/components/oculus-touch-controls.test.js

```diff
@@ -28,9 +28,9 @@ suite('oculus-touch-controls', function () {
 
       component.checkIfControllerPresent();
 
-      assert.notOk(injectTrackedControlsSpy.called);
-      assert.notOk(addEventListenersSpy.called);
-      assert.ok(removeEventListenersSpy.called);
+      this.sinon.assert.notCalled(injectTrackedControlsSpy);
```

Comment:
There is a series of these updates. We shouldn't be breaking the sinon abstraction on the spy and accessing called directly if we can avoid it. Instead use the helpers for notCalled, called, calledOnce, etc...